### DISCUSSION TITLE
Deployment on local network with traffic from AWS

### DISCUSSION THREAD

**Discussion by az0mb13** [Security-Onion-Solutions/securityonion#4082] [Unsupported Versions]
We are planning to set up Security Onion's management interface on our local network to monitor traffic from all the servers here, but we also need some traffic data from AWS to be fed into this. (Flow logs, Cloudtrail, etc)
How should I go about configuring this? Any suggestions on the deployment strategy and getting started would be really helpful.

**Comment by weslambert** [comment]
You can use something like OpenVPN to setup a VPN connection for a sensor in AWS to the local manager.

Otherwise, you can use the S3 input plugin for Logstash to pull this stuff in:
https://www.elastic.co/guide/en/logstash/current/plugins-inputs-s3.html

Last, we are working on having Filebeat modules easily enabled, which would also help with this in the near future.

**Reply by az0mb13** [reply]
Ah yes, that's exactly what I had in mind as well. Logstash plugin or a VPN.
Does the Security onion do any kind of anomaly detection for AWS-specific logs like there are some rules for them in Wazuh. Can I expect something similar after deployment?
